payments-core/src/main/java/com/stripe/android/payments/core/authentication/PaymentAuthenticator.kt

```diff
@@ -26,5 +30,19 @@ interface PaymentAuthenticator<Authenticatable> : ActivityResultLauncherHost {
         host: AuthActivityStarterHost,
         authenticatable: Authenticatable,
         requestOptions: ApiRequest.Options
+    ) {
+        val lifecycleOwner = host.lifecycleOwner
+        lifecycleOwner.lifecycleScope.launch {
+            lifecycleOwner.whenResumed {
+                performAuthentication(host, authenticatable, requestOptions)
+            }
+        }
+    }
+
+    @VisibleForTesting(otherwise = VisibleForTesting.PROTECTED)
+    abstract suspend fun performAuthentication(
```

Comment:
Can this be `protected`?

Reply:
Pull request updated, so this is now `protected`